### ISSUE TITLE
Can visit shipyard, hire crew, trade with wanderers before being able to communicate?

### ISSUE THREAD

**Issue by guillochon** [endless-sky/endless-sky#666]
It seems to me that all of these features should be disabled upon landing on the first Wanderer world, currently they are all available.

**Comment by Gkramme** [comment]
Hiring crew is a bit odd, but trading and veiwing ships are things that dont require talking.

**Comment by endless-sky** [comment]
At present there's no way to specify a planet that lets you refuel but does not let you trade, but I agree that eventually this should be changed.

**Comment by LocalGod79** [comment]
@jmhorjus i think that between us, we could tackle this for 0.9.0 - i believe that the true intent of this issue is:
Create increment/decrement-able flags for;
•Trading Post
•Job Board
•Bank
•"Employment office"
in addition to the spaceport/shipyard/outfitter, and adjust the map.txt if necessary (this is where i would come in).

in addition, a ui-invisible flag that would further increment/decrement refueling (independent of spaceport) seems to be desired.

**Comment by jmhorjus** [comment]
Agreed.  I can check out the areas of code involved and try to come up with a good design...probably will not get to this until Thursday evening though.

**Comment by TheGiraffe3** [comment]
At least if #9599 goes through, it may not be possible otherwise.
